src/track/beatmap.cpp

```diff
@@ -543,48 +532,47 @@ void BeatMap::translate(double dNumSamples) {
 }
 
 void BeatMap::scale(enum BPMScale scale) {
-
     QMutexLocker locker(&m_mutex);
     if (!isValid() || m_beats.isEmpty()) {
         return;
     }
 
     switch (scale) {
-    case DOUBLE:
-        // introduce a new beat into every gap
-        scaleDouble();
-        break;
-    case HALVE:
-        // remove every second beat
-        scaleHalve();
-        break;
-    case TWOTHIRDS:
-        // introduce a new beat into every gap
-        scaleDouble();
-        // remove every second and third beat
-        scaleThird();
-        break;
-    case THREEFOURTHS:
-        // introduce two beats into every gap
-        scaleTriple();
-        // remove every second third and forth beat
-        scaleFourth();
-        break;
-    case FOURTHIRDS:
-        // introduce three beats into every gap
-        scaleQuadruple();
-        // remove every second third and forth beat
-        scaleThird();
-        break;
-    case THREEHALVES:
-        // introduce two beats into every gap
-        scaleTriple();
-        // remove every second beat
-        scaleHalve();
-        break;
-    default:
-        DEBUG_ASSERT(!"scale value invalid");
-        return;
+        case DOUBLE:
```

Comment:
I will set IndentCaseLabels: false

Reply:
We do not have our own rule about case labels. If the Google default is to indent them, it is OK. 
It is just that I assume we have both in the code so "don't care would be the best option" to avoid noise.